install/installer/pkg/components/server/configmap.go

```diff
@@ -222,6 +223,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		},
 		ContentServiceAddr:           "content-service:8080",
 		ImageBuilderAddr:             "image-builder-mk3:8080",
+		UsageServiceAddr:             fmt.Sprintf("%s:%d", usage.Component, usage.GRPCServicePort),
```

Comment:
FYI, there's a utility called [`JoinHostPort`](https://pkg.go.dev/net#JoinHostPort) which can do this for you

Reply:
That's a tad nicer indeed - added.